### ISSUE TITLE
Auto-refresh missing since last update

### ISSUE THREAD

**Issue by dnwi** [gorhill/uMatrix#215]
Is there a way to get the auto-refresh option after making changes back? It was so convenient.

**Comment by alejandrolemus** [comment]
See https://github.com/gorhill/uMatrix/issues/198. From the release notes of version 0.9.0.0 (Other changes):

Smart reload is gone:
    Code was too complicated for what I consider a non-essential feature, compromising uMatrix's stability (my main goal).
    The code relied in part on behavior specific to Chromium.
    After using uBlock for weeks, months, I now favor having the user reload on his own when he decides so. I like it this way personally. You will quickly get used to it.
    The matrix can be stickied in the logger, rendering auto-reload irrelevant.
    The auto-reload code was not completely reliable anyway, it did not always accomplish its purpose (because point 1. and 2. above)

**Comment by mailinglists35** [comment]
this is very annoying, because there is a considerable distance on the screen between chrome reload button and umatrix icon that makes the whole process inconvenient.

the old behaviour was natural, logical and human. 
please bring it back, at any cost.

before: click click click on umatrix, tadaaa

after: click click click... move mouse to the opposite side, click, move mouse back, click, click click, move the mouse again five miles west, click, repeat.

**Comment by gorhill** [comment]
Duplicate of #198.
